5.0/en/0x50-V50-attacks-via-browser.md

```diff
@@ -43,48 +39,35 @@ note: tags and numbers for requirements are not changed, at the moment the goal
 
 | # | Description | L1 | L2 | L3 | CWE | Issue |
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
+| **50.1.1** | [ADDED] Verify that separate applications are hosted on different hostnames so as to benefit from the protections provided by the "same origin policy" and the hostname restrictions on cookies. | ✓ | ✓ | ✓ | 668 | [#1299](https://github.com/OWASP/ASVS/issues/1299) |
+| **14.4.8** | [ADDED, SPLIT FROM 14.5.3] Verify that the Cross-Origin Resource Sharing (CORS) Access-Control-Allow-Origin header uses a strict allow list of trusted origins. When "Access-Control-Allow-Origin: *" needs to be used, verify that the responses do not include any sensitive information. | ✓ | ✓ | ✓ | 183 |
```

Comment:
These two added from above

Reply:
Disagreement for both commented separately.

Reply:
Reverted